src/librustc/query/mod.rs

```diff
@@ -1068,6 +1068,11 @@ rustc_queries! {
             no_force
             desc { "codegen_unit" }
         }
+        query used_generic_parameters(key: DefId) -> BitSet<u32> {
```

Comment:
`parameters` here and elsewhere could probably be shortened to `params`.

Reply:
I've fixed this in dd24b5b.